src/components/Immersion/ImmersionPresentationList.js

```diff
@@ -51,11 +51,13 @@ function ImmersionPresentationList({ actions, isLoading, items }) {
                           <td className="px-6 py-4 text-fv-charcoal">
                             {english}
                           </td>
-                          <td className="text-right px-6">
+                          <td className="text-right px-6" aria-label="list">
```

Comment:
This aria-label was added to satisfy eslint requirements, and matches the use in `DictionaryListPresentation.js`